src/onegov/town6/forms/settings.py

```diff
@@ -118,20 +121,46 @@ class ChatSettingsForm(Form):
 
     enable_chat = BooleanField(
         label=_('Enable the chat'),
-        description=_('The chat is currently in an test-phase. '
-                      'Activate at your own risk.'),
         default=False
     )
 
+    specific_opening_hours = BooleanField(
+        label=_("Specific Opening Hours"),
+        description=_("If unchecked, the chat is open 24/7."),
+        fieldset=_("Opening Hours"),
+    )
+
+    opening_hours_chat = StringField(
+        label=_("Opening Hours"),
+        fieldset=_("Opening Hours"),
+        depends_on=('specific_opening_hours', 'y'),
+        render_kw={'class_': 'many many-opening-hours'}
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.time_errors = {}
+
     def process_obj(self, obj):
         super().process_obj(obj)
         self.chat_staff = obj.chat_staff or {}
         self.enable_chat = obj.enable_chat or {}
+        self.specific_opening_hours = obj.specific_opening_hours or {}
+        # self.opening_hours_chat.data = self.time_to_json(None)
+        if not obj.opening_hours_chat or None:
```

Comment:
What's the point of `or None`? Shouldn't this just be 

```suggestion
        if not obj.opening_hours_chat:
```

Reply:
True, this doesn't do anything.